inc/functions_calendar.php

```diff
@@ -130,7 +130,7 @@ function build_mini_calendar($calendar, $month, $year, &$events_cache)
 
 			$link_to_day = false;
 			// Any events on this specific day?
-			if(@count($events_cache["$day-$calendar_month-$calendar_year"]) > 0)
+			if((!empty($events_cache["$day-$calendar_month-$calendar_year"])) && @count($events_cache["$day-$calendar_month-$calendar_year"]) > 0)
```

Comment:
This should be:

```suggestion
			if(!empty($events_cache["$day-$calendar_month-$calendar_year"]) && @count($events_cache["$day-$calendar_month-$calendar_year"]) > 0)
```

Reply:
I guess it's down to coding style. I was trying to keep to how empty() has been used in MyBB already.

Reply:
In the line you linked to, those round brackets separate a complex condition (more than one possible passing result). In the line I am referring to in this PR, the round brackets serve no purpose and should therefore be removed. Please.